lib/vsc/administration/slurm/sacctmgr.py

```diff
@@ -39,11 +41,16 @@ class SacctMgrException(Exception):
     pass
 
 
+class SacctParseException(Exception):
+    pass
+
+
 class SacctMgrTypes(Enum):
     accounts = "accounts"
     users = "users"
     qos = "qos"
     resource = "resource"
+    activejobs = "activejobs"
```

Comment:
why only active jobs and not all jobs?

Reply:
jobs that have finished are not an issue. only queued and running.

Reply:
this is a generic module, i prefer to have one type for all jobs. activejobs are also jobs, so no reason to have this active jobs only

Reply:
ok

Reply:
done